plugins/lookup/dependent.py

```diff
@@ -144,7 +144,7 @@ def __evaluate(self, expression, templar, variables):
         ``variables`` are the variables to use.
         """
         templar.available_variables = variables or {}
-        expression = "{0}{1}{2}".format("{{", expression, "}}")
+        expression = '{{%s}}' % expression
```

Comment:
Here again the opening and ending sequence is part of the formatting string, which was intentionally not the case before. How about reverting to the original code?
```suggestion
        expression = "{0}{1}{2}".format("{{", expression, "}}")
```
Using `format()` is still totally legal and fine.

Reply:
No arguments :-)